src/Swift.Runtime/src/Swift/Protocols.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Swift;
+using Swift.Runtime;
+
+/// <summary>
+/// Represents Swift Hashable protocol in C#.
+/// </summary>
+public unsafe interface IHashable : ISwiftProtocol
```

Comment:
I would prefer `ISwiftHashable` I think that we should be explicit about this not being a `c#` interface

Reply:
I agree. Reverted from this PR.